monai/utils/misc.py

```diff
@@ -99,3 +102,48 @@ def process_bar(index, count, bar_len=30, newline=False):
     print(f"{index}/{count} {bar:s}  ", end=end)
     if index == count:
         print("")
+
+
+def manual_seed(seed):
+    _seed = seed
+
+
+def get_seed():
+    return _seed
+
+
+def set_determinism(enable=True, seed=2147483647, additional_settings=None):
```

Comment:
I think the function name should be `set_seed`, and it enables deterministic behaviour when `seed is not None`. so the expected behaviours would be:
- user runs `set_seed(seed=42)`, enables deterministic modules for all
- user does nothing, or runs `set_seed(seed=None)`, run all modules without deterministic modules

Reply:
updated in the latest patch.
Thanks.